presto-main/src/test/java/io/prestosql/connector/MockConnectorFactory.java

```diff
@@ -162,7 +173,7 @@ private MockConnector(
                 BiFunction<ConnectorSession, SchemaTableName, ConnectorTableHandle> getTableHandle,
                 Function<SchemaTableName, List<ColumnMetadata>> getColumns,
                 ApplyProjection applyProjection,
-                BiFunction<ConnectorSession, SchemaTableName, Optional<ConnectorNewTableLayout>> getInsertLayout,
+                ApplyTopN applyTopN, BiFunction<ConnectorSession, SchemaTableName, Optional<ConnectorNewTableLayout>> getInsertLayout,
```

Comment:
keep every argument on a separate line

```suggestion
                ApplyTopN applyTopN, 
                BiFunction<ConnectorSession, SchemaTableName, Optional<ConnectorNewTableLayout>> getInsertLayout,
```

Reply:
Done.